test.js

```diff
@@ -8,148 +7,188 @@ var chai = require('chai');
 var sinon = require('sinon');
 var sc = require('sinon-chai');
 var request = require('supertest');
+var logger = require('./index');
+var stripAnsi = require('strip-ansi');
 chai.use(sc);
 var expect = chai.expect;
 
 // test subjects
 var chalk = require('chalk');
-var app = require('./test-server');
-var log, sandbox;
+var app,
+  log,
+  sandbox;
 
 describe('koa-logger', function() {
-  beforeEach(function() {
-    sandbox = sinon.sandbox.create();
-    log = sandbox.spy(console, 'log');
-  });
 
-  afterEach(function() {
-    sandbox.restore();
-  });
+  describe('request logging', function() {
```

Comment:
Yeah, unfortunate, but because I needed to segment the existing test separately to the new test (in a new describe block) it was required. 😞